src/autometrics/tracker/prometheus.py

```diff
@@ -16,9 +17,17 @@
     COMMIT_KEY,
     VERSION_KEY,
 )
+
+from .exemplar import get_exemplar
+from .tracker import Result
 from ..objectives import Objective
 
 
+prometheus_client.REGISTRY.unregister(prometheus_client.GC_COLLECTOR)
```

Comment:
could you add a comment explaining these unregister calls? was this something we should've been doing all along? 

not sure if this is what you read, but [this thread](https://github.com/prometheus/client_python/issues/414) seemed relevant

Reply:
my bad, i set that when i was debugging exemplars but forgor 💀 to remove it. we probably shouldn't set these in autometrics.